index.html

```diff
@@ -322,6 +322,36 @@ <h3 class="skills_dropdown_title dropdown_child_class">
                   </li>
                </ul>
            </section>
+
+           <section class="contact" id="contact">
+            <div class="contact__container">
+              <h2 class="contact_header">Contact me</h2>
+              <p class="contact_description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+              </p>
+              <form action="#" method="post">
+                <ul>
+                  <li>
+                    <label for="name">Name:</label>
+                    <input type="text" id="name" name="firstname" placeholder="Enter your name here">
+                  </li>
+                  <li>
+                    <label for="email">Email:</label>
+                    <input type="text" id="email" name="firstname" placeholder="Enter your email here">
+                  </li>
+                  <li>
+                    <label for="email">Message:</label>
```

Comment:
- [ ] This label is incorrect as the element below is not an email, kindly fix this